handbook/people-ops/hiring.md

```diff
@@ -71,3 +71,16 @@ If any interviewer is asked by a candidate for interview feedback, they may resp
 ## Respecting time
 
 Interviewing requires time and effort, both for us and for candidates. The hiring manager is responsible for monitoring feedback during the interview phase, and if it becomes obvious that we won't be moving forward with a candidate, then the hiring manager should cancel any remaining interviews to be respectful of both the candidate's and interviewers' time.
+
+## Sharing company financials
+
+Choosing to join a company is a significant commitment. We want people to have an accurate view of Sourcegraph's financial and business position before they join. (Every Sourcegraph team member after they join has regular access to this information by viewing the board slides or asking the CEO directly.)
```

Comment:
```suggestion
Choosing to join a company is a significant commitment. We want people to have an accurate view of Sourcegraph's financial and business position before they join. (Every Sourcegraph team member after they join has regular access to this information by viewing the board slides or asking the CEO directly.)
```

Can you clarify that "asking the CEO directly" is not actually needed here? It's an odd addition, given all of our top-line forecasts/planning and results are already publicly available in GDrive. And anyone can get access to our full F/S by requesting them from the CEO _or_ anyone on the Ops team. My point is that this sentence makes it sound more secretive than it actually is!

Reply:
Fixed. Also added you as someone people can ask for the precise answers (not just me).